src/doc/unstable-book/src/library-features/asm.md

```diff
@@ -750,6 +750,9 @@ The supported modifiers are a subset of LLVM's (and GCC's) [asm template argumen
 | RISC-V | `reg` | None | `x1` | None |
 | RISC-V | `freg` | None | `f0` | None |
 | Hexagon | `reg` | None | `r0` | None |
+| PowerPC | `reg` | None | `0` | None |
+| PowerPC | `reg_nonzero` | None | `0` | `b` |
```

Comment:
Maybe choose a different register than 0 for `reg_nonzero`? ;)

Reply:
👀 Oops :P